readme.md

```diff
@@ -106,7 +116,21 @@ Then, make each route respond with a one-word string containing the RESTful acti
 var express = require("express");
 var app = express();
 
-// Your code starts here...
+app.get("/artists", function(req, res){
+  res.send("GET")
```

Comment:
OH. Oh. I was pretty sure I wasn't getting it right, this makes sense. Could I also have put "Read" here?